workshops/templates/workshops/organization.html

```diff
@@ -72,27 +72,26 @@
   </div>
 </div>
 
-<h3>Memberships</h3>
+<h2>Memberships</h2>
 {% if organization.membership_set.all %}
 <table class="table table-striped">
   <tr>
-    <th>Variant</th>
-    <th>Agreement start</th>
-    <th>Agreement end</th>
-    <th>Contribution type</th>
+    <th rowspan="2">Variant</th>
+    <th rowspan="2">Agreement start</th>
+    <th rowspan="2">Agreement end</th>
+    <th rowspan="2">Contribution type</th>
     <th colspan="3">Workshops w/o admin fee <span class="glyphicon glyphicon-question-sign" data-toggle="tooltip" title="Per year, counted for year of agreement start"></span></th>
     <th colspan="3">Self-organized workshops <span class="glyphicon glyphicon-question-sign" data-toggle="tooltip" title="Per year, counted for year of agreement start"></span></th>
     <th class="additional-links"></th>
   </tr>
   <tr>
-    <th></th><th></th><th></th><th></th>
     <th><small>Imposed</small></th><th><small>Completed</small></th><th><small>Remaining</small></th>
     <th><small>Imposed</small></th><th><small>Completed</small></th><th><small>Remaining</small></th>
     <th></th>
   </tr>
   {% for membership in organization.membership_set.all %}
   <tr>
-    <td>{{ membership.get_variant_display }}</td>
+    <td><a href="{{ membership.get_absolute_url }}">{{ membership.get_variant_display }}</a></td>
```

Comment:
Why not add a glyphicon to the right of the row to link to `membership_details` view?